docs/pages/community.md

```diff
@@ -23,12 +23,24 @@ Our [GitHub discussions forum][git-hub-discussions-forum] is a good place to cha
 
 You can find (and create) examples in the [Remix examples repository][the-examples-repository].
 
+## Community Runtimes
+
+You can find a list of Community Runtimes in the [@remix-run/server-runtime][remix-run-server-runtime] package.
+
+If you've written your own runtime, and think it will be valuable for others, you can update that list on GitHub.
```

Comment:
We should provide a link to the community runtimes as well here

```suggestion
If you've written your own runtime, and think it will be valuable for others, you can [update that list on GitHub][community-runtimes-github].
```

Reply:
I've reverted the whole list, see "runtimes" below